src/panels/profile/ha-long-lived-access-tokens-card.ts

```diff
@@ -0,0 +1,186 @@
+import { haStyle } from "../../resources/styles";
+import "../../styles/polymer-ha-style";
+import { HomeAssistant } from "../../types";
+
+@customElement("ha-long-lived-access-tokens-card")
+class HaLongLivedTokens extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+  
+  @property({ attribute: false }) public refreshTokens?: RefreshToken[];
+
+  protected render(): TemplateResult {
+    const accessTokens = this.refreshTokens
+      ?.filter((token) => token.type === "long_lived_access_token")
+      .reverse();
+
+    return html`
+      <ha-card
+        .header=${this.hass.localize(
+          "ui.panel.profile.long_lived_access_tokens.header"
+        )}
+      >
+        <div class="card-content">
+          ${this.hass.localize(
+            "ui.panel.profile.long_lived_access_tokens.description"
+          )}
+
+          <a
+            href="https://developers.home-assistant.io/docs/en/auth_api.html#making-authenticated-requests"
+            target="_blank"
+            rel="noreferrer"
+          >
+            ${this.hass.localize(
+              "ui.panel.profile.long_lived_access_tokens.learn_auth_requests"
+            )}
+          </a>
+          ${!accessTokens?.length
+            ? html`<p>
+                ${this.hass.localize(
+                  "ui.panel.profile.long_lived_access_tokens.empty_state"
+                )}
+              </p>`
+            : accessTokens?.map(
+                (token) => html`<ha-settings-row two-line>
+                  <span slot="heading">${token.client_name}</span>
+                  <div slot="description">
+                    ${this.hass.localize(
+                      "ui.panel.profile.long_lived_access_tokens.created",
+                      "date",
+                      relativeTime(
+                        new Date(token.created_at),
+                        this.hass.localize
+                      )
+                    )}
+                  </div>
+                  <ha-icon-button
+                    .token=${token}
+                    icon="hass:delete"
+                    @click=${this._deleteToken}
+                  ></ha-icon-button>
+                </ha-settings-row>`
+              )}
+        </div>
+
+        <div class="card-actions">
+          <mwc-button @click=${this._createToken}>
+            ${this.hass.localize(
+              "ui.panel.profile.long_lived_access_tokens.create"
+            )}
+          </mwc-button>
+        </div>
+      </ha-card>
+    `;
+  }
+
+  private async _createToken(): Promise<void> {
+    const name = await showPromptDialog(this, {
+      text: this.hass.localize(
+        "ui.panel.profile.long_lived_access_tokens.prompt_name"
+      ),
+    });
+
+    if (!name) {
+      return;
+    }
+
+    try {
+      const token = await this.hass.callWS<string>({
+        type: "auth/long_lived_access_token",
+        lifespan: 3650,
+        client_name: name,
+      });
+
+      setTimeout(() => {
```

Comment:
Why the timeout?

Reply:
Because the "close" event from the previous dialog kept force closing this one...

Reply:
Ah yeah 😄 that is annoying... But dont use setTimeout, uss `await new Promise((resolve) => setTimeout(resolve, 0));` (we should make a helper for that btw)

Reply:
Or just revert the previous screwup...